### PR TITLE
Add Online Playground of Programiz PRO for Python and C

### PR DESCRIPTION
**Pull request by awalrujaa**
## What does this PR do?
Add resource(s)

### Description
Add Online C Playground

This commit introduces an Online Python Playground and C Playground that allows users to write, edit, and run code directly in their browsers. This feature supports real-time coding and debugging, enhancing learning and development flexibility.

### Why is this valuable (or not)?
This allows users to write, edit, and run code directly in their browsers without the need of downloading or installation of other software. This saves time and helps you focus on learning. 
### How do we know it's really free?
Anybody can access it. It is an absolutely free resource.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
These additions appear to be duplicates. Is it just the url changing?

**Comment by awalrujaa** [comment]
No, Programiz Online Compiler and online Playground are two different things with different benefits for users.
While users can only run the code in Online Compiler, Programiz PRO's online playground is a platform where users can upload files, create projects, save, and share them for **free**.
Hence, the additions are not duplicates, rather these two are very different.
You can try Playground at: **[C Playground](https://programiz.pro/ide/c/ )** and **[Python Playground](https://programiz.pro/ide/python/9QJQ8BL19A)**
I hope this solves the confusion.

File: more/free-programming-playgrounds.md [hunk 1]
```diff
@@ -82,6 +82,7 @@
 
 ### <a id="c"></a>C
 
+* [C Playground - Online C Programming IDE](https://programiz.pro/ide/c)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:85]:
```suggestion
* [C Playground - Online C Programming IDE](https://programiz.pro/ide/c) - Programiz Pro
```

File: more/free-programming-playgrounds.md [hunk 2]
```diff
@@ -82,6 +82,7 @@
...
 * [InterviewBit - Online C Compiler IDE](https://www.interviewbit.com/online-c-compiler/)
 * [JDoodle - Online C Compiler IDE](https://www.jdoodle.com/c-online-compiler/)
 * [Programiz - Online C Compiler](https://www.programiz.com/c-programming/online-compiler/)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:88]:
```suggestion
* [Online C Compiler](https://www.programiz.com/c-programming/online-compiler/) - Programiz
```

File: more/free-programming-playgrounds.md [hunk 3]
```diff
@@ -313,6 +314,7 @@
 * [Programiz.com](https://www.programiz.com/python-programming/online-compiler/)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:314]:
```suggestion
* [Online Python Compiler](https://www.programiz.com/python-programming/online-compiler/) - Programiz
```

**Review by eshellman — CHANGES_REQUESTED** [review]
We try to use the title from the resource to more easily recognize duplicates

**Review by MEETDesai011 — APPROVED** [review]

**Comment by awalrujaa** [comment]
Thank you for approving the PR. When will this PR get merged in the main
branch?
Please let me know when it's merged.

Sincerely,
Ruja  Awal
 Computer Engineer
Nepal
[image: facebook icon] <https://www.facebook.com/ruja.awal/>[image:
linkedin icon] <https://www.linkedin.com/in/ruja-awal-b971551aa/>
IOE,

IOE, Pulchowk Campus

File: more/free-programming-playgrounds.md [hunk 4]
```diff
@@ -308,11 +309,12 @@
 * [Codepad](http://codepad.org/?lang=Python)
 * [InterviewBit - Online Python Compiler IDE](https://www.interviewbit.com/online-python-compiler/)
 * [Online Python](https://www.online-python.com)
+* [Online Python Compiler](https://www.programiz.com/python-programming/online-compiler/) - Programiz
 * [Online Python Compiler](https://www.tutorialspoint.com/online_python_compiler.php) - Tutorialspoint
 * [OnlineGDB](https://www.onlinegdb.com/online_python_compiler)
-* [Programiz.com](https://www.programiz.com/python-programming/online-compiler/)
 * [Pynative.com](https://pynative.com/online-python-code-editor-to-execute-python-code/)
 * [Python Online Compilers](https://www.codinguru.online/compiler/python)
+* [Python Playground](https://programiz.pro/ide/python) - by Programiz PRO
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:317]:
```suggestion
* [Python Playground](https://programiz.pro/ide/python) - Programiz PRO
```

File: more/free-programming-playgrounds.md [hunk 5]
```diff
@@ -82,9 +82,10 @@
 
 ### <a id="c"></a>C
 
+* [C Playground](https://programiz.pro/ide/c) - by Programiz PRO
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:85]:
```suggestion
* [C Playground](https://programiz.pro/ide/c) - Programiz PRO
```

**Review by eshellman — CHANGES_REQUESTED** [review]
almost there!

**Comment by awalrujaa** [comment]
@eshellman 
As per the change request, I've made the changes. Could you review?

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**